packages/react-native-codegen/src/parsers/__tests__/error-utils-test.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
```

Comment:
Can you enable Flow in this test?
```suggestion
 * @format
 * @flow strict-local
```